mlir/include/mlir/Conversion/Passes.td

```diff
@@ -381,6 +381,9 @@ def ConvertFuncToLLVMPass : Pass<"convert-func-to-llvm", "ModuleOp"> {
     Option<"useOpaquePointers", "use-opaque-pointers", "bool",
                        /*default=*/"true", "Generate LLVM IR using opaque pointers "
                        "instead of typed pointers">,
+    Option<"addComdatToLinkonceFuncs", "add-comdat-to-linkonce-funcs", "bool",
+                       /*default=*/"false", "Add an any comdat selector to Linkonce "
+                       "functions">,
```

Comment:
Can we get this from the data layout?

Reply:
Do you mean by adding a new entry to the Data Layout regarding comdats or that it should be implied by something already there?